lib/JSON/Validator.pm

```diff
@@ -1260,6 +1267,23 @@ block should return C<undef> on success and an error string on error:
 
 See L<JSON::Validator::Formats> for a list of supported formats.
 
+=head2 recursive_data_protection
+
+  my $jv = $jv->recursive_data_protections( $scalar );
+  my $current_state = $jv->recursive_data_protection;
+
+Recursive data protection is active by default, however it can be deactivated
+by assigning a false value to the C<recursive_data_protection> attribute.
```

Comment:
Use `L</recursive_data_protection>` instead.

Reply:
Switched to the link as suggested